_data/simulations/moose_1a_mpf/meta.yaml

```diff
@@ -0,0 +1,47 @@
+---
+benchmark:
+  id: 1a
+  version: 1
+data:
+- name: free_energy
+  url: https://raw.githubusercontent.com/mfrichtl/PFHub_benchmarks/main/problems/pfhub_benchmark_1.a1_csv.csv
```

Comment:
It doesn't really matter, but any particular reason for the "_csv.csv" file extension?

Reply:
@guyer -- Nope. MOOSE defaults to _output suffix and I just didn't bother changing it. I do agree it looks a bit goofy though.